### DISCUSSION TITLE
[BUG] init() not iterate over all FFTSizes[] array elements in display.cpp

### DISCUSSION THREAD

**Discussion by wlmqljj** [AlexandreRouma/SDRPlusPlus#1394] [Q&A]
I'm running master branch nightly build.  

I have set fftsize to 1024 in display section.   
And the config.json shows `"fftSize": 1024,`  
But every time I open a new instance,   
the fftsize in gui resets to 65536,   
and the fftsize in config.json remain the same.  
I'm not familiar with Cpp dev.  
Maybe It's a bug?  
[sdrpp.log.txt](https://github.com/AlexandreRouma/SDRPlusPlus/files/15197167/sdrpp.log.txt)
[config.json](https://github.com/AlexandreRouma/SDRPlusPlus/files/15197173/config.json)

With appreciation for any reply!

**Comment by wlmqljj** [comment]
Got it.  
Caused by not iterate over all FFTSizes[] array elements.  
https://github.com/AlexandreRouma/SDRPlusPlus/blob/2ea7ac496fe71b7a4862cabac7123ee4dc06b7d6/core/src/gui/menus/display.cpp#L33-L44
https://github.com/AlexandreRouma/SDRPlusPlus/blob/2ea7ac496fe71b7a4862cabac7123ee4dc06b7d6/core/src/gui/menus/display.cpp#L93-L99
Change to
##
        for (int i = 0; i < 10; i++) {
            if (fftSize == FFTSizes[i]) {
                fftSizeId = i;
                break;
            }
can fix the problem ;)

**Comment by AlexandreRouma** [comment]
These are discussion, not where bugs are reported. I moved it to the proper place: https://github.com/AlexandreRouma/SDRPlusPlus/issues/1396
